parse.go

```diff
@@ -132,6 +132,12 @@ type Option func(*Context)
 
 // WithConfigFile tells Parse to read the provided filename as a config file.
 // Requires WithConfigFileParser, and overrides WithConfigFileFlag.
+//
+// If you want both a config file flag and a default value when one is
+// unprovided, use WithConfigFileFlag and set your chosen default when
+// defining the flag. Because config files should generally be
+// user-specifiable, this option should be rarely used. Prefer
+// WithConfigFileFlag.
```

Comment:
This extra paragraph stutters. I'd prefer the comment block to read simply

```
// WithConfigFile tells Parse to read the provided filename as a config file.
// Requires WithConfigFileParser, and overrides WithConfigFileFlag.
// Because config files should generally be user-specifiable, this option
// should rarely be used. Prefer WithConfigFileFlag.
```

Reply:
sorry to make you spell it out for me, heh

fixed